product_default_image/models/product_template.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, tools
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    prevent_categ_change_image = fields.Boolean(
+        help='Used to prevent automatic changes to image fields when categ_id '
```

Comment:
I'm unclear on this. Why would we ever want the image fields to change, outside of the default_get?